apps/rates-service/src/internals/rates/UpdateDb.ts

```
@@ -114,52 +114,73 @@ export async function ensureLatestFxRate(now: number) {
+  const updated = await ensureLatestRates(now);
+  if (!updated) return false;
+
+  // Then sync those rates to the Oracle contract
+  await updateOracle(now);
+  return true;
 }
 
 export async function updateRates() {
+  const errors: Error[] = [];
+
   for (let i = 0; i < 5; i++) {
```

ESLint

[error] Unary operator '++' used.

(no-plusplus)